src/lib/ui/noise-sensor/NoiseSensorEventItem.tsx

```diff
@@ -39,38 +43,25 @@ export function NoiseSensorEventItem({
   )
 }
 
-function getContextLabel(event: Event): string {
-  switch (event.event_type) {
-    case 'noise_sensor.noise_threshold_triggered':
-      return `${t.noiseThresholdTriggered}`
-    case 'noise_detection.detected_noise':
-      return `${t.detectedNoise}`
-    default:
-      return t.noiseDetected
-  }
-}
-
 function getContextSublabel(event: Event): string | null {
-  if (
-    'noise_threshold_name' in event &&
-    typeof event.noise_threshold_name === 'string'
-  ) {
-    return event.noise_threshold_name
+  if ('noise_threshold_name' in event) {
+    return `${event.noise_threshold_name}`
```

Comment:
```suggestion
    return event.noise_threshold_name
```

Reply:
This is due to the properties being `unknown` on the Event type (since payload appears to be poorly typed and perhaps incorrect in some places). Working on a reasonable solution...

Reply:
Templating, while ugly, does work, however, as in the case below this.

Reply:
That's ok, we have better Event types coming! For now, you can add a `ts-expect-error` and file an `upstream` issue in this repo, e.g.,

```suggestion
    // @ts-expect-error <link to issue>
    return event.noise_threshold_name
```

Glad you pointed this out, since these look like minor "nits" but actually help us find gaps to ensure we don't return `undefined` in the UI :)

Reply:
Actually, just link https://github.com/seamapi/react/pull/633 :)